chore(release): version 2.0.5

mzrtamp opened the pull request:
After #604, every new commit will be thrown to this branch. No rebase needed?

Mednoob committed:
chore(deps): update lockfile

Mednoob committed:
feat(commands): add PurgeCommand

Mednoob committed:
fix(VolumeCommand): volume should be multiplied

Mednoob committed:
fix(AboutCommand): load value everytime command is used

Mednoob committed:
feat(MuteCommand): muting server owner shouldn't be allowed

Mednoob committed:
feat(AboutCommand): remove 'v' in node version

Mednoob committed:
chore: change version

Mednoob committed:
chore(lang/en): add 'commands.moderation.purge'

Mednoob committed:
chore(lang/en): add 'commands.moderation.mute.cantMuteOwner'

Commit:
chore(deps): bump @discordjs/voice from 0.7.2 to 0.7.3 (#602) (#603)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
chore(deps): upgrade lockfile

Commit:
chore(CODEOWNERS): change KurokuTetsuya to ZenShibata

Commit:
chore(*): compare changes (#605)

* chore(deps): bump @discordjs/voice from 0.7.2 to 0.7.3 (#602)

* chore(env): update notes grammaring (#604)

* chore(env): update notes grammaring

* chore(heroku): update string content

* chore(env): update notes grammaring

* chore(env): apply suggestion

* chore(env): apply suggestion

Co-authored-by: mzrtamp <mozartrafi@zhycorp.net>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: 21Z <69188068+21Z@users.noreply.github.com>

Commit:
chore(deps-dev): move @types/i18n and @types/node to devDependencies

Commit:
chore(deps): add ffmpeg-static

Commit:
Merge branch 'main' into feat/v2.0.5

Mednoob committed:
chore(deps): remove ffmpeg-static

Mednoob committed:
chore(deps): add prism-media@1.3.2

Mednoob committed:
feat(ClientUtils): add 'getFFmpegVersion' function

Mednoob committed:
refactor(AboutCommand): use 'ClientUtils#getFFmpegVersion' instead

Commit:
chore(deps): upgrade lockfile

Commit:
chore(SearchCommand): remove the selection message after cancellation

Mednoob committed:
chore(lang/en): change the 'commands.music.stayInQueue.247Disabled' message

Mednoob committed:
chore(lang/id): synchronize with 'lang/en'

Mednoob reviewed (approved):
LGTM. Just merge it.

dependabot[bot] committed:
chore(deps): bump @discordjs/voice from 0.7.3 to 0.7.5 (#607)

Bumps [@discordjs/voice](https://github.com/discordjs/voice) from 0.7.3 to 0.7.5.
- [Release notes](https://github.com/discordjs/voice/releases)
- [Changelog](https://github.com/discordjs/voice/blob/main/CHANGELOG.md)
- [Commits](https://github.com/discordjs/voice/compare/v0.7.3...v0.7.5)

---
updated-dependencies:
- dependency-name: "@discordjs/voice"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: mzrtamp <mozartrafi@zhycorp.net>

Commit:
chore(PingCommand): change 'Disconnected' to 'N/A'

Commit:
feat(commands): add InviteCommand

Commit:
chore(README): add youtube

Commit:
chore(*): update string content

Commit:
chore(isGlitch): don't remove devDependencies on non-glitch instance

Commit:
chore(config): tidy up file content

Commit:
chore(*): tidy up every ts file content

Contains string update, import line order sortings, etc.

Commit:
chore(lang/en): update some words

Commit:
chore(lang/id): menyinkronkan dengan 'lang/en'

Commit:
chore(lang/es): sincronizar con 'lang/en'
